skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -448,6 +449,160 @@ def _compute_metric_scores(
             return score[0] if score.size == 1 else score
         return score
 
+    @available_if(
+        lambda accessor: accessor._parent._timings_cache.get(("fit_time",)) is not None
+    )
+    def fit_time(self) -> float:
+        """Compute the time taken to fit the estimator.
+
+        Returns
+        -------
+        float
+            The fit time.
+
+        Examples
+        --------
+        >>> from sklearn.datasets import load_breast_cancer
+        >>> from sklearn.linear_model import LogisticRegression
+        >>> from sklearn.model_selection import train_test_split
+        >>> from skore import EstimatorReport
+        >>> X_train, X_test, y_train, y_test = train_test_split(
+        ...     *load_breast_cancer(return_X_y=True), random_state=0
+        ... )
+        >>> classifier = LogisticRegression(max_iter=10_000)
+        >>> report = EstimatorReport(
+        ...     classifier,
+        ...     X_train=X_train,
+        ...     y_train=y_train,
+        ...     X_test=X_test,
+        ...     y_test=y_test,
+        ... )
+        >>> report.metrics.fit_time()
+        ...
+        >>> report = EstimatorReport(
+        ...     classifier,
+        ...     fit=False,
+        ...     X_train=X_train,
+        ...     y_train=y_train,
+        ...     X_test=X_test,
+        ...     y_test=y_test,
+        ... )
+        >>> report.metrics.fit_time()
+        Traceback (most recent call last):
+        AttributeError: This '_MetricsAccessor' has no attribute 'fit_time'
+        >>> classifier.fit(X_train, y_train)
+        >>> report = EstimatorReport(
+        ...     classifier,
+        ...     X_train=X_train,
+        ...     y_train=y_train,
+        ...     X_test=X_test,
+        ...     y_test=y_test,
+        ... )
+        >>> report.metrics.fit_time()
+        Traceback (most recent call last):
+        AttributeError: This '_MetricsAccessor' has no attribute 'fit_time'
+        """
+        return self._parent._timings_cache[("fit_time",)]
+
+    def predict_time(
+        self,
+        *,
+        data_source: DataSource = "test",
+        X: Optional[ArrayLike] = None,
+        y: Optional[ArrayLike] = None,
+    ) -> float:
+        """Compute the time taken to compute predictions by the estimator.
```

Comment:
```suggestion
        """Renders the time taken to compute predictions by the estimator (in seconds).
```

Same as above, it's not actually computing anything.